python-data-version-control/src/prepare.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import pandas as pd
+
+folders_to_labels = {"n03445777": "golf ball", "n03888257": "parachute"}
+
+
+def get_filelist_and_labels(source_path, target_folders):
+    images = []
+    labels = []
+    for image_path in source_path.rglob("*/*.JPEG"):
+        filename = image_path.absolute()
+        folder = image_path.parts[-2]
+        if folder in target_folders:
+            images.append(filename)
+            label = folders_to_labels[folder]
```

Comment:
This is just to sync the `if` statement with other updates:

```suggestion
        if folder in FOLDERS_TO_LABELS:
            images.append(filename)
            label = FOLDERS_TO_LABELS[folder]
```